simpleimage/make_rootfs.sh

```diff
@@ -390,6 +423,78 @@ EOF
 		rm -f "$DEST"/etc/ssh/ssh_host_*
 		do_chroot ln -s /run/resolvconf/resolv.conf /etc/resolv.conf
 		;;
+	opensuse)
+		cat > "$DEST/etc/resolv.conf" << "EOF"
+nameserver 8.8.8.8
```

Comment:
Mhm this sounds strange and wrong. If DNS resolution is required during install, then copy the hosts resolv.conf as done with the other distros and clean it up when its no longer required and let the distribution do whatever DNS resolution it needs.